client/signup/steps/user/index.jsx

```diff
@@ -22,6 +22,19 @@ export default React.createClass( {
 		}
 	},
 
+	componentWillMount() {
+		let subHeaderText = this.props.subHeaderText;
+
+		/**
+		 * Update the step sub-header if they only want to create an account, without a site.
+		 */
+		if ( 1 === signupUtils.getFlowSteps( this.props.flowName ).length ) {
+			subHeaderText = this.translate( 'Welcome to the wonderful WordPress.com community' );
+		}
+
+		this.setState( Object.assign( {}, this.state, { subHeaderText: subHeaderText } ) );
```

Comment:
You don't need to use `Object.assign` here, as `setState` already performs a merge.